nginx/cernopendata.conf

```diff
@@ -22,16 +27,33 @@ server {
         root /usr/local/var/cernopendata/var/cernopendata-instance;
     }
 
-    location / {
-        proxy_pass http://web:5000;
+    location ~* /eos/opendata/.*\.(root|tgz|zip)$ {
+        proxy_cache off;
+        proxy_set_header X-Real-IP $remote_addr;
         proxy_set_header Host $host;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_pass https://eospublichttp.cern.ch;
+    }
+
+    # FIXME: Remember to activate cache for `/files` at some point.
```

Comment:
We can cache all files except`*.root`.

Or we can allow caching of `*.ig` (for event display), `*.csv`, `*.json`, `*.txt`...